Demo/src/main/java/com/braintreepayments/demo/ShopperInsightsFragment.kt

```diff
@@ -31,142 +53,224 @@ class ShopperInsightsFragment : BaseFragment() {
     private lateinit var payPalClient: PayPalClient
     private lateinit var venmoClient: VenmoClient
 
+    private val venmoLauncher: VenmoLauncher = VenmoLauncher()
+    private val paypalLauncher: PayPalLauncher = PayPalLauncher()
+
+    private lateinit var venmoPendingRequest: VenmoPendingRequest
+    private lateinit var paypalPendingRequest: PayPalPendingRequest
+
     // TODO: Refactor Shopper Insights to remove BraintreeClient
-//    override fun onCreateView(
-//        inflater: LayoutInflater,
-//        container: ViewGroup?,
-//        savedInstanceState: Bundle?
-//    ): View? {
-//        braintreeClient = getBraintreeClient()
-//        shopperInsightsClient = ShopperInsightsClient(braintreeClient)
-//
-//        venmoClient = VenmoClient(this, braintreeClient)
-//        venmoClient.setListener(this)
-//
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        braintreeClient = BraintreeClient(requireContext(), authStringArg)
+        shopperInsightsClient = ShopperInsightsClient(braintreeClient)
```

Comment:
Same thing here - let's instantiate `BraintreeClient` under the hood in `ShopperInsightsClient`